Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG PARENT_VERSION=2.1.2-node18.11.0
-ARG PORT=3000
-ARG PORT_DEBUG=9229
+ARG PORT=3008
+ARG PORT_DEBUG=9008
```

Comment:
I think this one should stay as `9229` so the default node port is used.  I'm not sure if there's anything else that would break if we change it.  So suggest keeping consistent with others internal to the pod.